cmd/cluster-node-tuning-operator/main.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"context"
+	"runtime"
+	"time"
+
+	stub "github.com/openshift/cluster-node-tuning-operator/pkg/stub"
+	sdk "github.com/operator-framework/operator-sdk/pkg/sdk"
+	k8sutil "github.com/operator-framework/operator-sdk/pkg/util/k8sutil"
+	sdkVersion "github.com/operator-framework/operator-sdk/version"
+
+	"github.com/sirupsen/logrus"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+)
+
+func printVersion() {
+	logrus.Infof("Go Version: %s", runtime.Version())
+	logrus.Infof("Go OS/Arch: %s/%s", runtime.GOOS, runtime.GOARCH)
+	logrus.Infof("operator-sdk Version: %v", sdkVersion.Version)
+}
+
+func main() {
+	printVersion()
+
+	sdk.ExposeMetricsPort()
+
+	resource := "tuned.openshift.io/v1alpha1"
+	kind := "Tuned"
+	namespace, err := k8sutil.GetWatchNamespace()
+	if err != nil {
+		logrus.Fatalf("failed to get watch namespace: %v", err)
+	}
+	resyncPeriod := time.Duration(5) * time.Second
```

Comment:
and we discussed lengthening this to 60s at some point right?

Reply:
Yes, in fact, I had 60 there previously.  5s is just a default generated by the operator SDK.

Reply:
Addressed by https://github.com/openshift/cluster-node-tuning-operator/pull/2